source/reference/minio-server/settings/root-credentials.rst

```diff
@@ -0,0 +1,102 @@
+.. _minio-server-envvar-root:
+
+===============
+`root` Settings
```

Comment:
Maybe this is a stray markdownism?
```suggestion
``root`` Settings
```

Alternatively, could just skip inline code format in this heading and go with `Root`.

Reply:
Yeah, going to omit the formatting. Monotext makes the headings and left nav look weird anyway.